examples/streamlit/stomata/README.md

```diff
@@ -8,7 +8,8 @@ Run VDP locally
 
 ```bash
 $ git clone https://github.com/instill-ai/vdp.git && cd vdp
-$ make all
+$ make build PROFILE=all  
+$ make dev PROFILE=all
```

Comment:
```suggestion
$ make all
```

We probably want to stick with released version for examples? @xiaofei-du @Phelan164

Reply:
Yes. This will be fixed in the `main` branch.